src/main/java/lindar/fonix/vo/KycResponse.java

```diff
@@ -32,13 +31,14 @@ public class KycResponse {
     private String  contractType;
 
     public static KycResponse from(InternalKycResponse internalKycResponse) {
+        Date statusDateTime = fonixDateUtil.getParsedDate("status time", internalKycResponse.getCompleted().getStatusTime(), internalKycResponse.getCompleted().getGuid());
```

Comment:
`internalKycResponse.getCompleted()` is repeated many times - maybe worth to extract it to a local variable?